docs/source/en/generation_strategies.md

```diff
@@ -20,11 +20,16 @@ A decoding strategy informs how a model should select the next generated token.
 
 This guide will help you understand the different decoding strategies available in Transformers and how and when to use them.
 
-## Greedy search
+> [!TIP]
+> `transformers` can also load custom generation strategies from the Hub! See the `Custom decoding methods` section below for instructions on how to load or share a custom generation strategy.
+
+## Basic decoding methods
```

Comment:
Reorganized the generation strategies docs to make it easier to find the docs regarding custom generation methods.

The outer level is now `basic decoding methods` (moved greedy decoding, sampling, and beam search here), `advanced decoding methods` (moved the other ones here), and `custom decoding methods` (new section for the code on the Hub)